### DISCUSSION TITLE
Why are some raylib source files created as .h only rather than .c/.h combo?

### DISCUSSION THREAD

**Discussion by IvanVnucec** [raysan5/raylib#3359] [Q&A]
Examples: [rcore.c](https://github.com/raysan5/raylib/blob/master/src/rcore.c) is .c and [rcamera.h](https://github.com/raysan5/raylib/blob/master/src/rcamera.h) is .h with functions in it...

**Comment by Peter0x44** [comment]
Because rcamera.h can be used standalone as a header-only library, outside of raylib
Or at least, the intent is to at least support that partly.
